server/src/main/resources/data.sql

```diff
@@ -0,0 +1,13 @@
+insert into feedback
+    (id, date, description, is_active, rest_id, user_name, counter_like, counter_dislike)
+values
+       (4,'2000-10-05', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud', 'true', 1, 'Don', 10 , 3),
+       (5,'2012-11-05', 'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. ', 'true', 1, 'Tom', 5 , 1),
+       (6,'2013-12-05', 'Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?', 'true', 1, 'Don', 10 , 20),
+       (7,'2015-11-05', 'But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete account of the system, and expound the actual teachings of the great explorer of the truth, the master-builder of human happiness.', 'true', 2, 'Tom', 6 , 2),
+       (8,'2015-02-05', 'Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.', 'true', 2, 'Tom', 7 , 20),
+       (13,'2000-04-05', 'At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa.', 'true', 2, 'Tom', 10 , 20),
+       (9,'1980-05-05', 'Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus.', 'true', 2, 'Don', 10 , 45),
+       (10,'2000-12-05', 'Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente delectus.', 'true', 3, 'Clark', 12 , 20),
+       (11,'2009-06-05', 'On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound to ensue.', 'true', 3, 'Devid', 10 , 205),
+       (12,'2010-10-05', 'In a free hour, when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business.', 'true', 3, 'Clark', 100 , 20);
```

Comment:
Let it name `feedback-data.sql`

Reply:
we need one file data for everyone. When you initials db you run one data and everything ok